kubernetes/resource_kubernetes_pod_security_policy.go

```diff
@@ -433,6 +447,20 @@ func resourceKubernetesPodSecurityPolicyRead(ctx context.Context, d *schema.Reso
 		return diag.FromErr(err)
 	}
 
+	sv, err := serverVersionGreaterThanOrEqual(conn, "1.21.0")
```

Comment:
Because this is in the create function, the user wont see this diagnostic until apply time. Is there a way we can get the warning to show on a fresh `terraform plan` using the resource?